src/resourcepacks/ResourcePackManager.php

```diff
@@ -49,6 +51,9 @@ class ResourcePackManager{
 	/** @var ResourcePack[] */
 	private array $uuidList = [];
 
+	/** @var array<string,string> */
```

Comment:
```suggestion
	/** @phpstan-var array<string, string> */
```
Please also add an `@var string[]`.

They need to be maintained this way because some tools don't support this generic style of phpdoc.